gallery/workload-insights/Azure Monitor.json

```diff
@@ -18,42 +18,12 @@
 			"id": "SQL",
```

Comment:
Remove the entire object with `id` `SQL`

Reply:
Yes I thought about it, I was not sure if we can remove it wholesale as called out by John. So left it as-is. If you think its ok to remove, I can